provider/resource_keycloak_realm.go

```diff
@@ -441,6 +441,148 @@ func resourceKeycloakRealm() *schema.Resource {
 				Type:     schema.TypeMap,
 				Optional: true,
 			},
+
+			// WebAuthn
+			"web_authn_policy": {
+				Type:     schema.TypeList,
+				Optional: true,
+				Computed: true,
+				Elem: &schema.Resource{
+					Schema: map[string]*schema.Schema{
+						"acceptable_aaguids": {
+							Type: schema.TypeList,
```

Comment:
you probably want this to be a `schema.TypeSet` instead.  the notable difference is that the order of elements within a `schema.TypeList` matters, so swapping the position of two items in a list would cause terraform to detect a change here.